src/redisvalue.rs

```diff
@@ -6,6 +6,7 @@ pub enum RedisValue {
     SimpleString(String),
     BulkString(String),
     BulkRedisString(RedisString),
+    Buffer(Vec<u8>),
```

Comment:
For consistency, I think we should call this `StringBuffer` instead of `Buffer`. We're talking about yet another way to represent a Redis String.